src/tasks/DSDFarmTask.py

```
@@ -242,8 +610,58 @@ def deside_combat_action(self):
+        self._apply_sound_config(dodge_action=self._dodge_without_direction)
+        with self.skip_sleep_checks() as skip:
+            skip.all = False
+            try:
+                deadline = time.time() + max_time
+                while time.time() < deadline:
+                    self.sleep(0.1)
+            except NotInCombatException:
+                self.log_info("dodge-only combat end (combat finished)")
+            except TaskDisabledException:
+                raise
+            except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)